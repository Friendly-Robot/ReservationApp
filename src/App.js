/*
* @flow
*/

import React from 'react';
import Reservations from './tabs/reservations';

type props = {};
export default class App extends React.PureComponent<props> {
  render() {
    return <Reservations />;
  }
}
