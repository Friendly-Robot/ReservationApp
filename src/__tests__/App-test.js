/**
 * @format
 */

import 'react-native';
import React from 'react';
import App from '../setup';

// fetch is not found globally and no fetcher passed, to fix pass a fetch for test environment.
global.fetch = require('node-fetch');

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
