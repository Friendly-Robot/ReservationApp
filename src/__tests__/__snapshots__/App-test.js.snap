// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgb(255, 255, 255)",
          "flexDirection": "row",
          "height": 65,
          "zIndex": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "rgb(30, 147, 242)",
            "fontSize": 16,
            "marginLeft": 15,
          }
        }
      >
        Reservations
      </Text>
    </View>
    <View
      accessibilityIgnoresInvertColors={true}
      style={
        Object {
          "height": 150,
          "width": 750,
          "zIndex": 2,
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/common/img/hotel-front-desk.jpeg",
          }
        }
        style={
          Array [
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "height": 150,
              "width": 750,
            },
            undefined,
          ]
        }
      />
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
          "paddingHorizontal": 25,
        },
        undefined,
      ]
    }
  >
    <ActivityIndicator
      animating={true}
      color="rgb(30, 147, 242)"
      hidesWhenStopped={true}
      size="large"
    />
    <Text
      style={
        Array [
          Object {
            "color": "rgb(30, 147, 242)",
            "fontSize": 14,
            "marginTop": 25,
          },
          undefined,
        ]
      }
    >
      Loading reservations
    </Text>
  </View>
</View>
`;
